backend/src/actions/docker/startContainerEventListener.ts

```diff
@@ -31,15 +32,36 @@ export default async function startContainerEventListener(datasource: GrafanaDat
       // Display event start in console.
       console.log('🚩 STARTED CONTAINER: ', event.Actor.Attributes.name, '!!');
 
-      // Invoke function to start metrics stream and create Grafana Dashboard
-      startStreamAndCreateDashboard(event.Actor.ID, event.Actor.Attributes.name, datasource);
+      // Invoke functions to start metrics stream and create Grafana Dashboard
+      startContainerMetricsStream(event.Actor.ID);
```

Comment:
NIT: instead of `event.Actor.ID` and `event.Actor.Attributes.name` in multiple places, it would be better for readability to create new variables `containerId` and `containerName` which are assigned to these.